config/main.py

```diff
@@ -69,6 +74,41 @@ def log_error(msg):
 # Helper functions
 #
 
+# Execute action on list of systemd services
+def execute_systemctl(list_of_services, action):
+    num_asic = _get_num_asic()
+    generated_services_list, generated_multi_instance_services = _get_sonic_generated_services()
+    if ((generated_services_list == []) and
+        (generated_multi_instance_services == [])):
+        log_error("Failed to get generated services")
+        return
+
+    for service in list_of_services:
+        if (service + '.service' in generated_services_list): 
+            try:
+                click.echo("Executing {} action on service {}...".format(action, service))
+                run_command("systemctl {} {}".format(action, service))
+            except SystemExit as e:
+                log_error("Failed to execute {} for service {} with error {}".format(action, service, e))
+                raise
+        elif (service + '.service' in generated_multi_instance_services):
```

Comment:
Do we have a case where a particular service is there in both in generated_services_list and generated_multi_instance_services list ? like may be database where we have a global database service and unit based database services for namespaces.

Reply:
currently there is no such service that is both global and multi-instance apart from database. But I have updated the logic to take care of this type of service, in case something like that gets added in future.